### PR TITLE
scheduler: fix inaccurate statistics and config

### PR DESCRIPTION
**Pull request by HunDunDM**
Signed-off-by: HunDunDM <hundundm@gmail.com>

### What is changed and how it works?

* scheduler: fix inaccurate statistics and config

### Check List

Tests

- Unit test

Code changes

Side effects

Related changes

- Need to cherry-pick to the release branch

### Release note

```release-note
None
```

### CODE REVIEW

**Review by nolouch — CHANGES_REQUESTED** [review]
lgtm

**Review by nolouch — APPROVED** [review]

File: server/schedulers/hot_region.go [hunk 1]
```diff
@@ -567,7 +578,18 @@ func (bs *balanceSolver) solve() []*operator.Operator {
...
 	}
 
-	if best == nil || !bs.sche.addPendingInfluence(op, best.srcStoreID, best.dstStoreID, infl) {
+	if best == nil {
+		return nil
+	}
+
+	var maxZombieDur time.Duration
+	switch {
+	case bs.rwTy == write && bs.opTy == transferLeader:
```
Comment (lhy1024) [inline server/schedulers/hot_region.go:587]: add some comments?

File: server/schedulers/utils.go [hunk 2]
```diff
@@ -220,24 +221,23 @@ func (lhs *Influence) add(rhs *Influence, w float64) *Influence {
...
-		origin: infl,
+		op:                op,
+		from:              from,
+		to:                to,
+		origin:            infl,
+		maxZombieDuration: maxZombieDur,
 	}
 }
 
-type storeLoad struct {
```
Comment (lhy1024) [inline server/schedulers/utils.go:237]: why change it?

**Review by lhy1024 — APPROVED** [review]

**Pull request merged**